x-pack/plugins/notifications/server/config/connectors_email_config.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, type TypeOf } from '@kbn/config-schema';
+import type { PluginConfigDescriptor } from '@kbn/core/server';
+
+export const configSchema = schema.object(
+  {
+    connectors: schema.maybe(
+      schema.object({
+        default: schema.maybe(
+          schema.object({
+            email: schema.maybe(schema.string()),
+          })
+        ),
+      })
+    ),
+  },
+  { defaultValue: {} }
+);
+
+export type ConnectorsEmailConfigType = TypeOf<typeof configSchema>;
+
+export const config: PluginConfigDescriptor<ConnectorsEmailConfigType> = {
+  schema: configSchema,
+};
```

Comment:
I wanted to have a "connectors specific part" so that we can more easily get rid of it if we ever change the underlying implementation. But I think it makes things unnecessarily complex. I'll merge both.